contrib/devtools/Makefile

```diff
@@ -122,6 +122,14 @@ ifeq (, $(shell which protoc-gen-swagger))
 else
 	@echo "protoc-gen-grpc-gateway already installed; skipping..."
 endif
+ifeq ($(UNAME_S),Linux)
+	@echo "Installing clang-format..."
+	sudo apt-get install clang-format
```

Comment:
Should we be using sudo in a script that will run on a devs machine? 

this `proto-tools` is getting bloated. Can we move this out of proto-tools?

Reply:
We can have individual targets, s.t. `proto-tools` is just a collection of those targets.

i.e.

```
proto-tools: proto-clang-format proto-buff ...
```

Reply:
Yes, having  individual targets looks better

Reply:
Our Makefiles _should not attempt_ to modify the user's operating systems packages set.
Plus, this works only on Debian and derivates. If I run Fedora and type `make` from the top-level directory then the build would fail and I would need to modify the Makefile in the `contrib/devtools` sub-directory to just make it work.

Please, please revert this change.

Reply:
I agree with @marbar3778. It does not make sense anymore to have a makefile for the proto toolset. We should migrate all this into a separate standalone shell script.